### ISSUE TITLE
[BUG] Live View of Rig and Cards has wrong Wallet address

### ISSUE THREAD

**Issue by smoochy** [jgulick48/NiceHashProfitabilityCalculator#34]
Hi,

The live view of the Rig Stats and the Card Stats has the wrong wallet address filled in by default. I assume it is your own? It is filled with 37s2ZBNJ64sa1bkYh3HWAkncJSxcXoBzvn, although the Graphs section has my correct wallet address filled in.

Steps to replicate issue:

- Start monitoring app. Put in any other wallet address **except** 37s2ZBNJ64sa1bkYh3HWAkncJSxcXoBzvn. 
- Start the Monitor
- Start the Viewer app. The Graphs section of Rig Stats and Card Stats do have the correct wallet

Expected behaviour: Live section also shows the entered wallet address.

Actual behaviour: Live view of wallet address 37s2ZBNJ64sa1bkYh3HWAkncJSxcXoBzvn is shown. Has to be changed manually

**Comment by jgulick48** [comment]
This should be and easy one to fix and I hope to have a new release out later today. Converted to Jira ticket https://jira.threegm.com/browse/NPC-16 where progress can be monitored.

**Comment by jgulick48** [comment]
Okay I did some more digging based on your description.  The first time you launch the application on a computer for the viewer it will have to have you enter your wallet address as it doesn't know which one to use to start off with.  I have removed the default address that was in there but the program does save it between sessions and will update all address fields if one of them is changed.  This should go our a little later today.

**Comment by jgulick48** [comment]
This change has been released with v1.0.0.74

**Comment by smoochy** [comment]
I need to bring this up again.

Updated to the new version. Started the monitor app. Set currency, power consumption and kWh price. I did not have been asked for my wallet address.

Started viewer app. Rig Graph has my wallet address shown up. Live section of Rig and Cards does not have my wallet address shown.
Put my wallet address into the field. Closed viewer app. Started viewer app again. My wallet address is shown in Graphs section, but again not in Live section.
